index.html

```diff
@@ -61,7 +61,7 @@
 
   <section id="getting-started">
     <aside>
-    <h3>Alright. How do I get started?</h3>
```

Comment:
@joshuaogle But you'd be wrong; please read the article I linked to in the original pull request. You're right that "already" is a standard English word, with a different meaning from "all ready". However, the analogy you proposed does not hold. There is no such word in standard English as "alright"; it is a spelling error.